grammar.json5

```diff
@@ -0,0 +1,121 @@
+//"field-func" : "function_name:(arguments,,, ...)"
```

Comment:
Consider adding the example part-by-part instead of at the end, such that the example of a specific option appears after it